ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -97,4 +81,27 @@ public Response streamQuery(KsqlRequest request) throws Exception {
       ));
     }
   }
+
+  private TopicStreamWriter getTopicStreamWriter(Map<String, Object> clientLocalProperties, PrintTopic statement) {
+    PrintTopic printTopic = statement;
```

Comment:
`final` params. Also, why assign the statement to the `printTopic`? looks redundant

Reply:
done